src/inmanta/server/server.py

```diff
@@ -1529,8 +1529,8 @@ def _recompile_environment(self, environment_id, update_repo=False, wait=0, meta
                                              stderr=process.Subprocess.STREAM,
                                              cwd=project_dir)
 
-            out, _, _ = yield [gen.Task(sub_process.stdout.read_until_close),
-                               gen.Task(sub_process.stderr.read_until_close),
+            out, _, _ = yield [sub_process.stdout.read_until_close(),
+                               sub_process.stderr.read_until_close(),
```

Comment:
In tornado 5 read_until_close already returns a future so the deprecated gen.Task is no longer required.

Reply:
perhaps add as comment to code?

Reply:
This is now more inline of what is expected. The comment should have been there before imho